cypress/commons/constants/generic/TestConstants.js

```diff
@@ -8,6 +8,7 @@ export const AUTH_QUERY_URL =
   'https://login.microsoftonline.com/e413b834-8be8-4822-a370-be619545cb49/oauth2/v2.0/token';
 export const LOCAL_WEBAPP_URL = 'http://localhost:3000';
 export const URL_POWERBI = `${LOCAL_WEBAPP_URL}/api/get-embed-info`;
+export const DEFAULT_WORKSPACE_ID = 'W-rXeBwRa0PM';
```

Comment:
Could we rename this constant to `BREWERY_WORKSPACE_ID` instead ? We're already using "default workspace" in the fixtures for a stubbed workspace with id "W-stbbdbrwry"